docs/source/sources.rst

```diff
@@ -27,4 +27,10 @@ Petri net extraction (:py:mod:`mira.sources.petri`)
 ---------------------------------------------------
 .. automodule:: mira.sources.petri
     :members:
-    :show-inheritance:
\ No newline at end of file
+    :show-inheritance:
+
+Aske net (:py:mod:`mira.sources.askenet`)
+-----------------------------------------
```

Comment:
We probably also need the submodules under this?

Reply:
Resolved in https://github.com/indralab/mira/pull/228/commits/12e1686bde739ff405962f7b9d53c779a7e84079